bbb_broker.php

```diff
@@ -665,11 +666,17 @@ function bigbluebuttonbn_broker_recording_ready($params, $bigbluebuttonbn) {
     }
     // Sends the messages.
     try {
-        bigbluebuttonbn_send_notification_recording_ready($bigbluebuttonbn);
+        // That if was for trying to workaround the several emails sent for one record ready
+        if (bigbluebuttonbn_get_count_callback_event_log($decodedparameters->record_id) == 0) {
+            bigbluebuttonbn_send_notification_recording_ready($bigbluebuttonbn);
+        }
+        $overrides = array('meetingid' => $bigbluebuttonbn->meetingid);
+        $meta = '{"recordID":'.$decodedparameters->record_id.'}';
+        bigbluebuttonbn_log($bigbluebuttonbn, BIGBLUEBUTTON_LOG_EVENT_CALLBACK, $overrides, $meta);
         header('HTTP/1.0 202 Accepted');
     } catch (Exception $e) {
-        $error = 'Caught exception: '.$e->getMessage();
-        header('HTTP/1.0 503 Service Unavailable. '.$error);
+          $error = 'Caught exception: '.$e->getMessage();
```

Comment:
Why did you change the identation?, htat should have triggered a cedechecker warning. Did you check the code with codechecker?